pkg/registry/corekind/base_gen.go

```diff
@@ -54,6 +57,11 @@ func (b *Base) Playlist() *playlist.Kind {
 	return b.playlist
 }
 
+// Preferences returns the [kindsys.Interface] implementation for the preferences kind.
```

Comment:
off-topic: I briefly read `kindsys` as `kidneys` and that's gonna amuse me all day

Reply:
oh my god i cannot unsee this and i love it